scripts/centos-req.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sudo yum groupinstall -y "Development tools"
+sudo yum install -y gmp-devel mpfr-devel libmpc-devel zlib-devel vim git java java-devel
+curl https://bintray.com/sbt/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-sbt-rpm.repo
+sudo yum install -y sbt texinfo gengetopt
+sudo yum install -y expat-devel libusb1-devel ncurses-devel cmake "perl(ExtUtils::MakeMaker)"
+# deps for poky
+sudo yum install -y python36 patch diffstat texi2html texinfo subversion chrpath git wget
+# deps for qemu
+sudo yum install -y gtk3-devel
```

Comment:
Just to propose a (non-essential) alternative: `gtk3-devel` is overkill for a headless system if not using the GUI.
The minimum dependencies for QEMU are:
```suggestion
sudo yum install -y glib2-devel libfdt-devel pixman-devel zlib-devel
```
Source: https://wiki.qemu.org/Hosts/Linux